documentation/ztp-for-factories/ztp-install-factory-edge-pipeline.adoc

```diff
@@ -153,9 +153,9 @@ edgeclusters:
 <2> This value must match `master0`, `master1` or `master2`.
 <3> Optional: Interfaces to ignore in the host.
 <4> NIC connected to the external DHCP.
-<5> NIC connected to the internal network.
+<5> NIC connected to the internal network (This interface is optional).
 <6> MAC address for the NIC connected to the external DHCP network.
-<7> MAC address for the NIC connected to the internal network
+<7> MAC address for the NIC connected to the internal network (This MAC address is optional).
```

Comment:
I would probably add a footnote 'is optional when using VLAN tag for dualnic'

Reply:
This is not to use vlan for both interfaces...only one of these will be tagged (the internal interface). The external is ethernet without vlan tag

Reply:
sorry I got your point of view...let's update it

Reply:
changed to explain mac is only optional if nic also is not present